src/_P097_Esp32Touch.ino

```diff
@@ -150,25 +197,30 @@ boolean Plugin_097(uint8_t function, struct EventStruct *event, String& string)
             if (touched) {
               if (P097_SEND_TOUCH_EVENT) {
                 // schedule a read to update output values and send to controllers
+                UserVar.setFloat(event->TaskIndex, 1, 1);
```

Comment:
To make that state really useful it should not be dependent on the TOUCH/RELEASE_EVENT settings. If the release event is not enabled (the default) it would never be reset.

Reply:
You are right. But this also raises the question of whether it is still useful to have these checkboxes (touch and release) at all. Duration is the only one I would keep.

Reply:
Well, both Touch and Release cause the same event, so being able to ignore Release can be useful, and we try to keep stuff backward compatible where/when possible, so please don't remove them...